Dockerfile-ngc-pytorch-hpc

```diff
@@ -0,0 +1,28 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+# Copy various shell scripts that group dependencies for install
+# Then, make sure that these scripts have execute permissions
+COPY dockerfile_scripts /tmp/det_dockerfile_scripts
+RUN chmod -R 777 /tmp
```

Comment:
definitely it is not ok to make all of `/tmp` rwx.  Can you make the individual files that you intend on executing executable instead?